test/pages/lifetimeallowance/MultipleBenefitCrystallisationEventPageSpec.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.lifetimeallowance
+
+import models.{CheckMode, NormalMode}
+import pages.behaviours.PageBehaviours
+
+class MultipleBenefitCrystallisationEventPageSpec extends PageBehaviours {
+
+  "MultipleBenefitCrystallisationEventPage" - {
+
+    beRetrievable[Boolean](MultipleBenefitCrystallisationEventPage)
+
+    beSettable[Boolean](MultipleBenefitCrystallisationEventPage)
+
+    beRemovable[Boolean](MultipleBenefitCrystallisationEventPage)
+  }
+
+  "normal mode navigation" - {
+
+    "when user doesn't have multiple public sector BCE " in {
```

Comment:
Should the navigation differ based on the answer? 

Irrespective, there should be test for both cases. What is the significance of the answer to this question therefore?